src/main/java/net/mcreator/element/converter/fv32/ArmorSpecialInformationConverter.java

```diff
@@ -86,6 +86,6 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 	}
 
 	@Override public int getVersionConvertingTo() {
-		return 31;
+		return 32;
```

Comment:
We are at fv32 currently, so it needs to be updated to fv33, the same way as everything else in this PR.

Reply:
Why? Is a snapshot with fv32 out? No, then I can keep the same fv, what's wrong with that?

Reply:
Each PR should bump FV version, developers also use workspaces and ignoring this will break workspaces of developers